homeassistant/components/elevenlabs/config_flow.py

```diff
@@ -50,8 +48,17 @@ async def get_voices_models(
     """Get available voices and models as dicts."""
     httpx_client = get_async_client(hass)
     client = AsyncElevenLabs(api_key=api_key, httpx_client=httpx_client)
-    voices = (await client.voices.get_all()).voices
-    models = await client.models.get_all()
+    try:
+        voices = (await client.voices.get_all()).voices
+    except ApiError as exc:
+        _LOGGER.error("Failure retrieving voices: %s", exc)
```

Comment:
Please undo all the extra exception logging. Those will be handled by the code that called this code.

Reply:
All removed.